src/TextLink/__tests__/__snapshots__/index.test.js.snap

```diff
@@ -1,63 +1,3 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`TextLink type: primary, size: default, newTab: false Should match snapshot 1`] = `
-<a
-  className="jsx-952623663"
-  href="https://kiwi.com"
-  onClick={
-    [MockFunction] {
-      "calls": Array [
-        Array [],
-      ],
-    }
-  }
->
-  My text link
-  <JSXStyle
-    css="a.__jsx-style-dynamic-selector{font-size:14px;color:#00a991;font-weight:500;-webkit-text-decoration:none;text-decoration:none;cursor:pointer;}a.__jsx-style-dynamic-selector:hover{-webkit-text-decoration:underline;text-decoration:underline;color:#00a991;}a.__jsx-style-dynamic-selector:focus{outline-width:3px;}"
-    dynamic={
-      Array [
-        "14px",
-        "#00a991",
-        "500",
-        "none",
-        "underline",
-        "#00a991",
-      ]
-    }
-    styleId="211793505"
-  />
-</a>
-`;
-
-exports[`TextLink type: secondary, size: large, newTab: true Should match snapshot 1`] = `
-<a
-  className="jsx-3896007344"
-  href="https://kiwi.com"
-  onClick={
-    [MockFunction] {
-      "calls": Array [
-        Array [],
-        Array [],
-      ],
-    }
-  }
-  target="_blank"
->
-  My text link
-  <JSXStyle
-    css="a.__jsx-style-dynamic-selector{font-size:16px;color:#171b1e;font-weight:500;-webkit-text-decoration:underline;text-decoration:underline;cursor:pointer;}a.__jsx-style-dynamic-selector:hover{-webkit-text-decoration:underline;text-decoration:underline;color:#00a991;}a.__jsx-style-dynamic-selector:focus{outline-width:3px;}"
-    dynamic={
-      Array [
-        "16px",
-        "#171b1e",
-        "500",
-        "underline",
-        "underline",
-        "#00a991",
-      ]
-    }
-    styleId="211793505"
-  />
-</a>
-`;
+exports[`TextLink should match snapshot 1`] = `null`;
```

Comment:
the test probably cannot find anything and snap it, can you check it @williamkolmacka ?